frontend/resourceadm/components/ConsentPreview/ConsentPreview.tsx

```diff
@@ -165,7 +151,7 @@ export const ConsentPreview = ({
                 <StudioHeading level={3} data-size='2xs'>
                   {resourceName[language]}
                 </StudioHeading>
-                <Markdown>{texts.resourceText}</Markdown>
+                <div data-testid='consentPreviewMarkdown'>{transformText(texts.resourceText)}</div>
```

Comment:
Maybe you want to add the testId in the `testids.js` file for reusability? 😄 
`frontend/testing/testids.js`